### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by 300ms**
Errors fixed and merged from testing branch ^^

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Reviewing......

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi guys good start on this project, I'll be pointing out some corrections.

### Intuit products
-  The products row is under the Intuit logo and above the form.
- It shows the Mint logo and text (logo on the left, text on the right).
-  It shows the Quickbooks logo and text (logo on the left, text on the right).
- It shows the Turbotax logo and text (logo on the left, text on the right).

##### Form
## Top
* [ ] It has a title `Create an Intuit account`.
* [ ] It has a description: `One account for everything Intuit, including Mint`.
* [ ] It has a `Learn More` link.

Also, the goal of this project is to replicate the original webpage, the current implementation does not do that please compare implementation with the original design

File: index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                <a id="brand-img" href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D34758894225416457621264800979794955614%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1565729615%26ivid%3D024d5df0-3ed3-47c7-b0f7-a500be1696a9#"></a>
+            </div>
+            <div class="add-links">
+                <a href="#"><img class="Color" src="img1.png" alt=""></a>
+            </div>
+            <div class="add-links2">
+                <a href="#"><img class="Color" src="img2.png" alt=""></a>
+            </div>
+            <form class="form" action="#" method="post">
+                <fieldset class="fieldset">
```
Comment (Oghenebrume50) [inline index.html]: This fieldset tag is not really needed here, what exactly are we grouping :thinking:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Please fix these things and submit for re-review, feel free to chat me up on slack if there are issues

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi, team, the changes are good but we need to do a little more work, the screenshot above shows how your form displays and what is expected. Please pay attention to colours, size positioning. Basically replicate the design, good job so far, I am sure you guys can do it :+1:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi good job on the corrections, however somethings were missed  
- The intuit logo was not added :smiley: 
- The key lock icon in the password field is still off, please align properly.
- The input fields and the button should be of same width, there should be equal space between them and the container on both sides, that is, they should be centered.

File: stylelint.config.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    } ],
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always-single-line",
+    "at-rule-semicolon-newline-after": "always",
+    "block-closing-brace-empty-line-before": "never",
+    "block-closing-brace-newline-after": "always",
+    "block-closing-brace-newline-before": "always-multi-line",
+    "block-opening-brace-newline-after": "always-multi-line",
+    "block-opening-brace-space-after": "always-single-line",
+    //"block-opening-brace-space-before": "always",
```
Comment (Oghenebrume50) [inline stylelint.config.js:21]: Please delete commented code, if this is affecting your development, just delete other than commenting it out

File: stylelint.config.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    "media-feature-range-operator-space-before": "always",
+    "media-query-list-comma-newline-after": "always-multi-line",
+    "media-query-list-comma-space-after": "always-single-line",
+    "media-query-list-comma-space-before": "never",
+    "no-eol-whitespace": true,
+    "no-missing-end-of-source-newline": true,
+    "number-leading-zero": "always",
+    "number-no-trailing-zeros": true,
+    "property-case": "lower",
+    /*"rule-empty-line-before": [ "always-multi-line", {
```
Comment (Oghenebrume50) [inline stylelint.config.js:82]: Please delete commented code

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Make these changes and submit for re-review :innocent:

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi guys! everything looks almost fine. :raised_hands: :medal_sports: 
#### There are some observations:

- [x] Your html code is not passing the validation test, ensure that it does.
 
- [x] The intuit logo is not showing up, check it out. :microscope: :smile: 

#### Here is what your site looks like, there's no **Intuit logo**
Screenshot from 2019-08-31 00-46-30


#### Here is what the intuit site looks like
Screenshot from 2019-08-31 00-46-13

#### Here's another observation, ensure that this sentence ``` One account for everything Intuit. Learn more``` is in a single line.


### Implement these changes and submit another review form, you are almost there :fire: :muscle:

**Comment by 300ms** [comment]
Code passes from HTML validation now. The logo is visible and positioned, I don't know why you can't see it, I added a screenshot here-exists in the README.md file too-, so you can see the logo exists and "Learn More" anchor text is in a new line.  
(Captured with Chrome extension--Full Page Screen Capture--)
screencapture-file-C-Users-ShADo-Desktop-Microverse-html-forms-index-html-2019-09-01-06_52_21

**Review by bolah2009 — APPROVED** [review]
# Nice Job @300ms 

### Good Job implementing all suggested changes. It's a :+1: from me...

#### The [clone](https://raw.githack.com/300ms/html-forms/98aabfc8e835e9d7d2bb188300a97a6dd7105840/index.html) looks good to me, uses HTML forms elements effectively. :ok_hand: 

[OPTIONAL but strongly recommended] Refactor the code to keep asset (CSS or images) files in a separate directory. Currently, all files are in the root folder which may make it difficult to find a specific type of file especially when they become many.


Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)  

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**